src/fights.ts

```diff
@@ -311,7 +313,7 @@ const freeFightSources = [
     () => get("questL11Worship") !== "unstarted" && get("_drunkPygmyBanishes") === 10,
     () => {
       putCloset(itemAmount($item`bowling ball`), $item`bowling ball`);
-      retrieveItem(10 - get("_drunkPygmyBanishes"), $item`Bowl of Scorpions`);
+      retrieveItem(11 - get("_drunkPygmyBanishes"), $item`Bowl of Scorpions`);
```

Comment:
this was on purpose - the last fight is in the saber code, it's a little awkward but i think this is the best way to do it?

Reply:
The first 10 fights are on lines 275-283, the last fight with saber code is on lines 283-308, and then this part on lines 310-316 seem to be in the case you didn't have a saber.

In this case the macro would fail, because you fetched 10-10=0 bowl of scorpions.